src/mn_setup.c

```diff
@@ -1121,116 +1119,158 @@ static void DrawGyroCalibration(void)
 //
 // killough 8/15/98: rewritten
 
-static void DrawInstructions()
+typedef enum
+{
+    MENU_HELP_OFF,
+    MENU_HELP_AUTO,
+    MENU_HELP_KEY,
+    MENU_HELP_PAD
+} menu_help_t;
+
+static menu_help_t menu_help;
+
+static void DrawInstructions(void)
 {
-    static char joyb_buf[64];
     int index = (menu_input == mouse_mode ? highlight_item : set_item_on);
     const setup_menu_t *item = &current_menu[index];
     const int flags = item->m_flags;
 
-    if (ItemDisabled(flags) || print_warning_about_changes > 0)
+    if (menu_help == MENU_HELP_OFF || print_warning_about_changes > 0)
     {
         return;
     }
 
     // There are different instruction messages depending on whether you
     // are changing an item or just sitting on it.
 
-    const char *s = "";
+    char s[80];
```

Comment:
Please initialize it to 0 for the case of an empty string:
```c
char s[80] = {0};
```
or 
```c
char s[80];
s[0] = '\0';
```

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1890/commits/c3f84bf2c69eab31c49fa85fb04809405869fceb